docs/chapters/baseplatform/building-PELUX-sources.rst

```diff
@@ -172,29 +172,34 @@ Dependencies:
           <https://docs.docker.com/engine/installation/linux/docker-ce/debian/>`_
           to install the latest version of Docker.
 
-Procedure:
-^^^^^^^^^^
+Full build procedure:
+^^^^^^^^^^^^^^^^^^^^^
 
 1. Clone the pelux-manifests git repository with submodule
 
 .. code-block:: bash
 
-    git clone --recurse-submodules git@github.com:Pelagicore/pelux-manifests.git
+    git clone --recurse-submodules https://github.com/Pelagicore/pelux-manifests.git
     cd pelux-manifests
 
 
 2. Build and run Docker container
 
+.. note:: If image was build previously you can reuse it and omit build command
+          To verify image existence please run 'docker images pelux'
+
 .. code-block:: bash
 
     docker build --build-arg userid=$(id -u) --build-arg groupid=$(id -g) -t pelux .
-    docker run -t -d --name pelux-build -v $(pwd):/docker pelux /bin/bash
+    docker run -t -d --name pelux-build -v $(pwd):/docker -w /docker pelux /bin/bash
 
 3. Run inside the Docker container
 
 At this point, we recommend using ``docker exec pelux-build <command>`` and to follow the same
 instructions as when building locally (but inside the Docker container).
 
+.. note:: You need ~40GB of free space in working directory
```

Comment:
This is very dependent the variant, but also the more time you build a new image, the more space is required since each artifact is timestamped so if the image is 5GB then the required amount of space increases with 5GB per build.

I would recommend at least 100 GB but 50 GB may be the absolute minimum, so consider to change the text to something like "You need >50GB of ..."